packages/workflow/src/internals.ts

```diff
@@ -416,9 +425,56 @@ export class State {
     [
       '__enhanced_stack_trace',
       (): EnhancedStackTrace => {
-        // const stacks = this.getStackTraces();
-        // const sourceMap = this.sourceMap;
-        return { locations: [], sources: {} };
+        const sourceMap = this.sourceMap;
+        const stacks = this.getStackTraces().join('\n\n');
+        const sdkInfo: SDKInfo = { name: 'typescript', version: '' };
+        const locationsPaths: FileLocation[] = [];
+        const sourceMapRecord: Record<string, FileSlice[]> = {};
+        const fileSliceSize = 50; //Lines
+        const parsedStacks = errorStackParser.parse({ name: '', message: '', stack: stacks });
+
+        parsedStacks.forEach(
+          (stackTraceEntry: { columnNumber: any; lineNumber: any; fileName: string | undefined }) => {
```

Comment:
Why `any`?

Reply:
Will change it to
`(stackTraceEntry: StackFrame) => {
            if (stackTraceEntry.columnNumber && stackTraceEntry.lineNumber && stackTraceEntry.fileName) {`

Reply:
Probably don't even need to specify the type, TS should be able to infer it.